bin/omarchy-notification-status

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
```

Comment:
Think we're going to need to come up with something modular here. Where not everything we need is hardcoded into a single file. I think?

Reply:
Definitely, I'm testing and running benchmarks now, this is the very first draft. The file name is also not right.